database/database_queries.py

```diff
@@ -17,12 +17,15 @@ def get_table(table):
 
 # patient_info returns command to get all info about one patient for a specified table
 def patient_info(tablename, patientid):
-    return f"SELECT * FROM {tablename} WHERE PatientID={patientid}"
+    return f"SELECT * FROM {tablename} WHERE PatientID='{patientid}'"
 
 # type_patient_info returns command to get all info about a particular type for a patient
 def type_patient_info(tablename, patientid, entrytype):
-    return f"SELECT * FROM {tablename} WHERE PatientID={patientid} AND Type='{entrytype}'"
+    return f"SELECT * FROM {tablename} WHERE PatientID='{patientid}' AND Type='{entrytype}'"

```

Comment:
@dlhi This gave me straight ebola for an hour till I figured out you needed to add in those damn quotes

You tested with "1141" not "lsfknlkdsng=dfgfsd-gdsfgs-gfds-gfsd"

that is all